services/queue/src/artifacts.js

```diff
@@ -379,6 +534,36 @@ let replyWithArtifactDownload = async function({ taskId, runId, name, req, res,
   res.set('x-taskcluster-artifact-storage-type', storageType);
 
   // Handle S3 artifacts
```

Comment:
This comment ("Handle S3 artifacts") should be further down in the code next to the s3 handling. It could probably be removed, as it is pretty obvious from the code.